### PR TITLE
🐛 [PRTL-1360] filter pie aggregations with all filters

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: src/packages/@ncigdc/containers/RepositoryPage.js [hunk 1]
```diff
@@ -265,6 +267,8 @@ export const RepositoryPageQuery = {
           cases {
             aggregations(filters: $filters aggregations_filter_themselves: false) {
               ${CaseAggregations.getFragment('aggregations')}
+            }
+            pieAggregations: aggregations(filters: $filters aggregations_filter_themselves: true) {
```
Comment (shanewilson) [inline src/packages/@ncigdc/containers/RepositoryPage.js]: this is how it is done on the exploration page? just want to make sure they are consistent

Comment (Jephuff) [inline src/packages/@ncigdc/containers/RepositoryPage.js]: yep, but they are called `pies` there, I'll update to match.

Comment (shanewilson) [inline src/packages/@ncigdc/containers/RepositoryPage.js]: there was also a bug where the label would say 1 files0 or 1cases0 if there was only 1 item in the pie

Comment (shanewilson) [inline src/packages/@ncigdc/containers/RepositoryPage.js]: probably a typo

**Comment by shanewilson** [comment]
there was also a bug where the label would say 1 files0 or 1cases0 if there was only 1 item in the pie
probably a typo

**Comment by Jephuff** [comment]
yep! found it, it was `0` when it should have been `''`'. Is there a ticket for that?

**Comment by shanewilson** [comment]
I don't think so but it was found at the same time this bug was so I guess they can be fixed together

**Pull request merged**